modules/widgets/social-media-icons.php

```diff
@@ -325,6 +325,12 @@ public function remove_username( $val ) {
  * @return void
  */
 function wpcom_social_media_icons_widget_load_widget() {
+	// [DEPRECATION]: Only register widget if active widget exists already
+	$has_widget = is_active_widget( false, false, 'wpcom_social_media_icons_widget', false );
```

Comment:
Are we sure this works?  Testing if a widget is active before registering it?

Also, it seems like this may be more efficient to cache in a transient rather than having WordPress scan through every sidebar on every page load.  That's probably a blocker.